### PR TITLE
Add support for integrated_services

### PR DESCRIPTION
**Pull request by ika108**

### CODE REVIEW

File: command/project-create.go [hunk 1]
```diff
@@ -35,6 +35,23 @@ func (c *ProjectCreateCommand) Run(args []string) int {
 	dbEngine := c.flagSet.String("db-engine", "", "Select database engine")
 	dbSize := c.flagSet.String("db-size", "", "Select database size")
 	dbVersion := c.flagSet.String("db-version", "", "Select database version")
+
+	consulEnabled := c.flagSet.Bool("consul", false, "Enable Consul")
```
Comment (obourdon) [inline command/project-create.go:39]: Like most other "features" have a `feat_enabled` pattern, I think that same should apply for consul nomad and vault

File: command/project-create.go [hunk 2]
```diff
@@ -116,6 +133,21 @@ func (c *ProjectCreateCommand) Run(args []string) int {
 		payload["slack_webhook"] = *slackURL
 	}
 
+	var integrated_services []map[string]string
+
+	if *consulEnabled {
+		integrated_services = append(integrated_services, map[string]string{"name": "consul", "enabled": "true", "basicauth": *consulBasicAuth, "prefix": *consulPrefix, "ipwhitelist": *consulIpWhiteList})
```
Comment (obourdon) [inline command/project-create.go:139]: I am not sure anymore on how empty strings are treated on the web side as opposed to no corresponding section but if I remember the UI/front part this should be ok but still to be verified

**Review by obourdon — CHANGES_REQUESTED** [review]

**Review by obeyler — CHANGES_REQUESTED** [review]
As consul and nomad  are hosted by same host may ip white list should be set only once ?

**Comment by obourdon** [comment]
> As consul and nomad are hosted by same host may ip white list should be set only once ?

I do not think this is a good idea, first because I can think of some cases where people will not want to access those 2 services from the same origins, second because this will introduce a discrepancy with what is possible within the front/UI.

CLI should be a perfect symetry/equivalent with Front/UI

File: command/project-create.go (hunk 1 above)
Comment (ika108) [reply command/project-create.go:39]: The only instance of such case i can find is `backup-enabled` in db-set.go 
Should i rename my flag to something in the like of `consul-enabled` ?

thanks for your insight :)

**Review by obourdon — APPROVED** [review]

**Pull request merged**